sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
```

Comment:
Might be a good idea to add a description what the function does before this heading instead of starting with what it returns.

Reply:
Yes,I thought so. Then it returns the same thing as it is described as,so changed it. Although,now I have changed it.